test/unit/backend/storage_rewrite_test.rb

```diff
@@ -12,7 +12,7 @@ class StorageRewriteTest < ActiveSupport::TestCase
       [service.metrics.hits, backend_api.metrics.hits].each do |metric|
         ::BackendMetricWorker.expects(:perform_now).with(service.backend_id, metric.id)
       end
-      StorageRewrite.rewrite_provider(provider.id)
+      StorageRewrite.new.rewrite_provider(provider.id)
```

Comment:
This is broken, will be fixed (and new tests added) when the implementation is finalized.